README.md

```diff
@@ -60,7 +60,20 @@ dism { 'Internet-Explorer-Optional-amd64':
 ~~~
 
 ##Reference
-
+###Deployment on Azure Windows Server 2012 R2 Datacentre (Image from Azure Store)
+####Windows Server 2012 R2 Datacentre does not have dism installed under c:\windows\sysnative 
+####it is under c:\windows\system32
```

Comment:
@danieloa While we are waiting on someone from docs to review this, would you mind reviewing http://docs.puppetlabs.com/puppet/latest/reference/lang_windows_file_paths.html#summary and determining if the reason you don't have a sysnative is due to not being subjected to FileSystem Redirection (something built into Windows 64-bit systems)? 

If you are running a 64-bit version of Puppet, then this would be the case.